readme.md

```diff
@@ -1,32 +1,40 @@
-# Lighthouse  [![Build Status](https://travis-ci.org/GoogleChrome/lighthouse.svg?branch=master)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://coveralls.io/repos/github/GoogleChrome/lighthouse/badge.svg?branch=master)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master)
+# Lighthouse  [![Build Status](https://travis-ci.org/GoogleChrome/lighthouse.svg?branch=master)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://coveralls.io/repos/github/GoogleChrome/lighthouse/badge.svg?branch=master)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
 
-> Lighthouse analyzes web apps and web pages, collecting modern performance metrics and insights on developer best practices.
-
-**Lighthouse requires Chrome 56 or later.**
+> [Lighthouse](https://developers.google.com/web/tools/lighthouse/) analyzes web apps and web pages, collecting modern performance metrics and insights on developer best practices.
 
 ## Installation
 
+_Lighthouse requires Chrome 56 or later._
+
 ### Chrome extension
 
-[Install from the Chrome Web Store](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)
+[Install from the Chrome Web Store](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk).
 
-### Node CLI [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
+### Node CLI
 
-**Requires Node v6+.**
+_Lighthouse requires Node 6 or later._
 
 ```sh
 npm install -g lighthouse
-# or use yarn:
 # yarn global add lighthouse
 ```
 
 ## Running Lighthouse
 
+### Chrome DevTools
+
+As of Chrome 60 or later, Lighthouse is integrated directly into the Chrome DevTools.
```

Comment:
i like this too. i'm equally happy doing this in a followup PR since we're already hitting a lot here.